portfolio/src/main/webapp/script.js

```diff
@@ -52,3 +52,64 @@ function prev(){
 
 var myVar = setInterval(next, 4000);
 
+
```

Comment:
As far as I can tell, your html is only using the `getMessageUsingArrowFunctions` implementation. It's surprising to me that other implementations are here as well. If you want to keep them around for reference, I'd suggest either commenting them out or putting them at the end of the file and clearly marking them with a big comment explaining why they're there. You could also just remove them.